recipes/configure.rb

```diff
@@ -14,7 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'runit'
+
```

Comment:
Well, you won't get runit installed if you remove 'include_recipe runit" ;) That's the reason of /usr/bin/sv not found